tests/test_invalid_nixpkgs_package.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+set -e
+
+ln -s tests/invalid_nixpkgs_package/workspace.bazel WORKSPACE
+ln -s tests/invalid_nixpkgs_package/nested-build.bazel BUILD
+ln -s tests/invalid_nixpkgs_package/default.nix default.nix
+
+# We need to provide a nixpkgs to create an output store path which is
+# a folder (because nixpkgs_package requires the output store path to
+# be a directory).
+#
+# We use the absolute path of the static busybox mkdir tool in the
+# builder script of the hello derivation. Since building this
+# derivation doesn't rely on nixpkgs, we can easily relocate the
+# /nix/store path in the Bazel sandbox.
+sed "s;BUSYBOX-ABS-PATH;${PWD}/external/busybox_static/bin/;g" -i default.nix
+
+# This is a create all directories required to run Nix locally.
+mkdir -p ${TEST_TMPDIR}/nix/{store,var/nix,etc/nix}
+export NIX_REMOTE="local"
+export NIX_STORE_DIR=${TEST_TMPDIR}/nix/store
+export NIX_STATE_DIR=${TEST_TMPDIR}/nix/var/nix
+export NIX_LOG_DIR=${TEST_TMPDIR}/nix/var/log/nix
+export NIX_CONF_DIR=${TEST_TMPDIR}/nix/etc/nix
+
+
+# First, we build a script outputing "hello world 1". This message
+# string comes from the file message.nix which is a dependency of the
+# nixpkgs_package rule.
+echo '"hello world 1"' > message.nix
+bazel build //:hello-output --sandbox_writable_path /dev
```

Comment:
Why is the `-sandbox_writable_path /dev` part needed?

Reply:
No, it was actually not required. Sorry.